docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+---
+id: contributing
+title: Contributing to the Doc Hub
+slug: /contributing
+---
+
+The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to modify an existing document or create a new one.
+
+This page will walk you through how to edit an existing document or create a new one, how to display docs in the sidebar where desired, and how to make these changes in GitHub.
+
+Documents are written in Markdown (`.md`) files using [GitHub-flavored Markdown syntax](https://github.github.com/gfm/) and are compiled into static web pages. [MDX](https://mdxjs.com/) is also supported, meaning you may embed React components in content as needed. Refer to Docusaurus [Markdown features](https://v2.docusaurus.io/docs/markdown-features) for more technical information about these Markdown pages.
+
+<div className="flex">
+<div>
+<b>Notifications</b>
+
+If you would like to subscribe to notifications for this repository, including when someone is requesting doc changes with a pull request that you can review and approve, be sure to "Watch" the repository.
+
+Also, see [configuring notifications](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications) in GitHub.
+
+Alternatively, consider subscribing to this repository in the [GitHub Slack](https://slack.github.com/) app by searching for it under "Apps" inside Slack.
+
+</div>
+   <img src="../img/github-watch.png" alt="GitHub Watch" style={{ width: '200px', marginLeft: '16px' }} />
+</div>
+
+## GitHub Workflow
+
+All documentation edits and additions will require a pull request into the `main` branch because merged changes will trigger an automatic deployment, but **fear not if you don't prefer checking into code - you probably don't need to.\***
+
+\*If you'd like to preview your sidebar changes or if you need to preview how a document containing MDX will build, you'll likely want to check into your branch and run `yarn start` to see the changes. Please reach out to the Experimenter team if you need help.
+
+### GitHub UI
+
+The GitHub user interface allows users to make file changes for those that would prefer not to check into the code repository via CLI. See [the GitHub UI Docs](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository) if you need further details than what's provided here.
```

Comment:
Maybe, `If you aren't as familiar with git, we recommend using the GitHub user interface to make changes to docs.`